app/views/categories/_form.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_with do |form| %>
```

Comment:
why not `form_for`?

Reply:
in form_for we have to mention the object or model name because it takes the model as an instance. form_with do with or without the model instance.

Reply:
we should use form_for. The errors will be set on the object, and will be easy to show

Reply:
what happened to this

Reply:
I never call this partial. I think I made this by mistake.